src/components/input/SelectNext.tsx

```diff
@@ -166,6 +238,13 @@ export type SelectProps<T> = CompositeProps & {
 
   'aria-label'?: string;
   'aria-labelledby'?: string;
+
+  /**
+   * Test seam.
+   * Used to determine if current browser supports native popovers.
+   * Defaults to `'popover' in document.body`
```

Comment:
How the component determines whether native popovers are supported can be an implementation detail, so this last line isn't needed.

Also, perhaps `usePopover` or `useNativePopover` would be a more accurate name, since this prop determines whether the Popover API is used, not whether the browser actually supports it.

Reply:
Good point.

Reply:
I'm going with `listboxAsPopover`, because `use...` sounds like a hook (for a moment I thought your suggestion was going in that line)